### PR TITLE
Grids & Slider components

### PR DESCRIPTION
**Pull request by MarkVed17**
~ Added grids component
~ Added range slider component
~ Here are the previews:
https://deploy-preview-10--raptor-ui.netlify.app/components/grid/grid.html
https://deploy-preview-10--raptor-ui.netlify.app/components/slider/slider.html

### CODE REVIEW

**Review by horsemaker — APPROVED** [review]
Looks clean & perfect to me. 👌

File: components-list.html [hunk 1]
```diff
@@ -1,160 +1,156 @@
...
+  <p class="page-title">Components</p>
+
+  <!-- Alerts -->
+  <main class="main-content">
+    <a href="components/alert/alert.html" class="component-card">
+      <div class="component-name">
+        <span>Alerts</span>
+      </div>
+      <div class="img-container">
+        <img src="/assets/alerts.svg" style="width: 224px; height: 96px" class="component-card-img" />
```
Comment (pritam-kr) [inline components-list.html:39]: remove inline CSS

File: components-list.html [hunk 2]
```diff
@@ -1,160 +1,156 @@
...
+      </div>
+    </a>
+
+    <!-- Avatars -->
+    <a href="components/avatar/avatar.html" class="component-card">
+      <div class="component-name">
+        <span>Avatars</span>
+      </div>
+      <div class="img-container">
+        <img src="/assets/avatars.svg" style="width: 138px; height: 100px" class="component-card-img" />
```
Comment (pritam-kr) [inline components-list.html:49]: also put an alt attribute  with some text

File: components-list.html [hunk 3]
```diff
@@ -1,160 +1,156 @@
...
+      </div>
+      <div class="img-container">
+        <img src="/assets/slider.svg" style="width: 224px; height: 115px" class="component-card-img" />
+      </div>
+    </a>
+
+  </main>
+</body>
+
+</html>
```
Comment (pritam-kr) [inline components-list.html:156]: break line here.

File: components/slider/slider.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const sliderInputs = document.querySelectorAll('.slider-input input');
+const sliderRanges = document.querySelectorAll('.slider-range input');
+const progressBar = document.querySelector('.slider-range .progress');
```
Comment (pritam-kr) [inline components/slider/slider.js:3]: I think to use an ID selector to select elements.

**Review by pritam-kr — APPROVED** [review]
I have commented on some minor changes.
but remove inline CSS as Vikram sir said.

File: components-list.html (hunk 1 above)
Comment (MarkVed17) [reply components-list.html:39]: Yes, you’re right. But in this case I’ll have to add separate classes for every component to just alter the dimensions of images which isn’t efficient imo.

File: components-list.html (hunk 2 above)
Comment (MarkVed17) [reply components-list.html:49]: Yes, thanks for pointing this out.

**Pull request merged**